style.css

```diff
@@ -0,0 +1,33 @@
+body {
+  display: flex;
+  flex-direction: row;
```

Comment:
we don't have to write this property as the flex-direction default value is a row 👍

Reply:
- [ ] kindly make your reviews polite and less authoritative

Reply:
Thank you for all 🙏